magento/magento2
# Fix a PHP error being thrown on a Magento error being thrown

### Description (*)

I ran into a small problem with the product media (because my pub/media wasn't writable). But I did not get the actual error that is thrown, I got a PHP error because 'directory' was not set (as you can see by my fix).
I thought, why not have my first contribution be a simple fix.

PHP: 7.4.15
Magento: 2.4.2

### Related Pull Requests

-

### Fixed Issues (if relevant)

-

### Manual testing scenarios (*)

1. Have a simple magento shop with X (might as well be 1) product *including* images
2. Change the pub/media to incorrect permissions/owners
3. Try to visit the product detail page in the backend

Screenshot of the PHP Notice in stead of the actual error being thrown (also in the exception.log)
![image](https://user-images.githubusercontent.com/1510476/115877711-0ac1b680-a448-11eb-8f33-6fdb50cd47a7.png)

### Questions or comments

-

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32819: Fix a PHP error being thrown on a Magento error being thrown
